apps/comps/types/enums.ts

```diff
@@ -1,8 +1,6 @@
-export enum CompetitionStatus {
-  Pending = "pending",
-  Active = "active",
-  Ended = "ended",
-}
+import type { CompetitionStatus as CompetitionStatusDB } from "@recallnet/db/repositories/types";
+
+export type CompetitionStatus = CompetitionStatusDB;
```

Comment:
note: i forgot that we have these legacy types in `apps/comps`, and i noticed this one is inaccurate (it is missing the `ending` value). fixed it here.

i'm not sure what our best practice is...should `packages/db` be the source of truth for types, and we import them in `apps/comps` etc? 

https://linear.app/recall-labs/issue/ENG-874/refactor-appscomps-types-to-use-package-exported-types

Reply:
Yea thanks for the issue. I think the answer is the comps app should just use inferred types and when needed import types from services or db. Is this type even used?